lib/cc/engine/analyzers/javascript/parser.js

```diff
@@ -1,15 +1,51 @@
-var fs = require('fs');
-var esprima = require('esprima');
-var source = "";
+var babel = require('babel');
 
 process.stdin.resume();
-process.stdin.setEncoding('utf8');
+
+var source = "";
+var toScrub = [
+  'leadingComments',
+  'trailingComments',
+  'shadow',
+  'start',
+  'end',
+  'raw',
+  'rawValue'
+];
 
 process.stdin.on('data', function(chunk) {
   source += chunk;
 });
 
 process.stdin.on('end', function() {
-  var result = esprima.parse(source, { loc: true });
-  process.stdout.write(JSON.stringify(result));
+  var ast = babel.transform(source).ast;
+  var program = ast.program;
+  console.log(
+    JSON.stringify(format(program), null, 2)
```

Comment:
Oops, we don't even need those anymore since we don't care about pretty printing the output.